e2e/wdio/core/tests/step-input.e2e-spec.ts

```diff
@@ -0,0 +1,159 @@
+import { StepInputPo } from '../pages/step-input.po';
+import {
+    refreshPage,
+    getElementArrayLength,
+    click,
+    getValue,
+    doubleClick,
+    pause,
+    getText,
+    clearValue,
+    setValue,
+    getElementClass,
+    sendKeys,
+    isElementDisplayed
+} from '../../driver/wdio';
+import { sections } from '../fixtures/appData/step-input-content'
+
+describe('Time component test', function () {
+    const stepInputPage = new StepInputPo();
+    const {
+        formExample,
+        labelExample,
+        localExample,
+        stateExample,
+        configExample,
+        defaultExample,
+        currencyExample,
+        step,
+        input,
+        plusButton,
+        minusButton,
+        text,
+        textForDisabledExample
+    } = stepInputPage;
+
+    beforeAll(() => {
+        stepInputPage.open();
+    }, 1);
+
+    afterEach(() => {
+        refreshPage();
+    }, 2);
+
+    it('should check default example inputs', () => {
+        for (let i = 0; i < sections.length; i++) {
+            checkInputWorking(sections[i], 'click', 'positive')
+            checkInputWorking(sections[i], 'click', 'negative')
+            checkInputWorking(sections[i], 'type', 'negative')
+            checkInputWorking(sections[i], 'type', 'positive')
+        }
+    })
+
+    it('should check limitation for step-input', () => {
+
+        setValue(configExample + input, '10', 5);
+        expect(getElementClass(configExample + plusButton, 5)).toContain('is-disabled')
+        setValue(configExample + input, '-10', 5);
+        expect(getElementClass(configExample + minusButton, 5)).toContain('is-disabled')
+
+    })
+
+    it('should verify that in specific input step is 0.5', () => {
```

Comment:
I think you could just do setValu to 0 and hit + button and expect the new value to be 0.5
or
get startValue
click + 
expect newValue to be startValue + 0.5

Reply:
should also check the `-` step